<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageFoc</class>
 <widget class="QWidget" name="PageFoc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="ParamTable" name="generalTab"/>
       </item>
       <item>
        <widget class="DetectFoc" name="detectFoc" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Sensorless</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="ParamTable" name="sensorlessTab"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Hall Sensors</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="ParamTable" name="hallTab"/>
       </item>
       <item>
        <widget class="DetectFocHall" name="detectFocHall" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Encoder</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="ParamTable" name="encoderTab"/>
       </item>
       <item>
        <widget class="DetectFocEncoder" name="detectFocEncoder" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>HFI</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="ParamTable" name="hfiTab"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="ParamTable" name="advancedTab"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ParamTable</class>
   <extends>QTableWidget</extends>
   <header>widgets/paramtable.h</header>
  </customwidget>
  <customwidget>
   <class>DetectFoc</class>
   <extends>QWidget</extends>
   <header>widgets/detectfoc.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DetectFocHall</class>
   <extends>QWidget</extends>
   <header>widgets/detectfochall.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DetectFocEncoder</class>
   <extends>QWidget</extends>
   <header>widgets/detectfocencoder.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
